extension/src/telemetry/uuid.ts

```diff
@@ -40,30 +40,31 @@ const writeMissingConfigs = (
 }
 
 const readOrCreateConfig = (): string | undefined => {
-  const dvcAppDir = getDVCAppDir()
-  const iterativeAppDir = getIterativeAppDir()
+  try {
```

Comment:
[F] This is called by `getUserId` which is in turn called here:

```
  reporter.sendTelemetryEvent(
    eventName as string,
    { ...sanitizedProperties, user_id: getUserId() },
    measurements
  )
```

☝🏻 this is essentially a `noop` if the user has disabled telemetry but the above code is being called regardless